nessie/src/main/java/org/apache/iceberg/nessie/TableReference.java

```diff
@@ -62,6 +62,7 @@ public static TableReference parse(TableIdentifier path) {
   /**
    * Convert dataset read/write options to a table and ref/hash.
    */
+  @SuppressWarnings("StringSplitter")
```

Comment:
Rather than suppress the warning can we fix it? I believe Guava's `Splitter` is used in other places